android/sdl_android/src/main/java/com/smartdevicelink/managers/lockscreen/LockScreenManager.java

```diff
@@ -379,23 +379,17 @@ private void downloadDeviceIcon(final String url){
 		new Thread(new Runnable(){
 			@Override
 			public void run(){
-				try{
-					if(mLockScreenDeviceIconManager.isIconCachedAndValid(url)) {
-						DebugTool.logInfo("Image Is Up To Date");
-						deviceLogo = mLockScreenDeviceIconManager.getFileFromCache(url);
-						if (deviceLogo == null) {
-							deviceLogo = AndroidTools.downloadImage(url);
-							mLockScreenDeviceIconManager.saveFileToCache(deviceLogo, url);
-						}
-					} else {
-						DebugTool.logInfo("Lock Screen Icon Update Needed");
-						deviceLogo = AndroidTools.downloadImage(url);
-						mLockScreenDeviceIconManager.saveFileToCache(deviceLogo, url);
+				mLockScreenDeviceIconManager.retrieveIcon(url, new LockScreenDeviceIconManager.OnIconRetrievedListener() {
```

Comment:
The indentation seems off on these

Reply:
this is the removed code, the new code seems aligned correctly with the run method above to me.

Reply:
I just pulled your latest updates, it still appears incorrect on the new code. They appear to have an extra tab of indentation.